src/lib/components/Header/Header.test.ts

```diff
@@ -7,7 +7,8 @@ import Header from "./Header.svelte";
 describe("Header", () => {
   it("renders", () => {
     render(Header);
-    expect(screen.getByText("<Project title>")).toBeVisible();
+    // Implicit ARIA role for the HTML <header> element
+    expect(screen.getByRole("banner")).toBeVisible();
```

Comment:
This test failed with my changes, so this is both a fix and a change to something that should be more future-proof. I also tried to follow the [recommended precedence for queries by Testing Library](https://testing-library.com/docs/queries/about/#priority)